model/docs/example_with_datasets.qmd

```diff
@@ -302,15 +302,94 @@ out = hosp_model.plot_posterior(
 )
 ```
 
+We can use [ArviZ](https://www.arviz.org/) to visualize the results. Convert the fitted model to Arviz InferenceData object:
+```{python}
+# | label: convert-inferenceData
+import arviz as az
+idata = az.from_numpyro(hosp_model.mcmc)
+```
+Below we plot $90 \%$ and $50 \%$ highest density interval for latent hospital admissions using [plot_hdi](https://python.arviz.org/en/stable/api/generated/arviz.plot_hdi.html):
+
+```{python}
```

Comment:
Suggestion:
Title: Posterior Hospital Admissions
Y-axis: Hospital

It might be cool if we eventually had a helper function to plot multiple credible intervals based on a vector of widths.

Reply:
I agree! I can work on that if we decide on it.